synapse/handlers/oidc_handler.py

```diff
@@ -1047,10 +1072,22 @@ def parse_config(config: dict) -> JinjaOidcMappingConfig:
                     % (e,)
                 )
 
+        extra_attributes = {}  # type Dict[str, Template]
+        if "extra_attributes" in config:
+            for key, value in config["extra_attributes"].items():
```

Comment:
typically we make config settings handle `None` gracefully, thus:

```suggestion
        for key, value in (config.get("extra_attributes") or {}).items():
```